frontend/sw360-portlet/src/main/webapp/html/search/view.jsp

```diff
@@ -153,7 +153,17 @@
         result.push({
             "DT_RowId": '${doc.id}',
             "0": '${doc.type}',
-            "1": "<sw360:DisplaySearchResultLink searchResult="${doc}"/>"
+            <core_rt:choose>
+                <core_rt:when test="${doc.type.equals('project')
+                                   || doc.type.equals('component')
+                                   || doc.type.equals('release')
```

Comment:
I think that maybe (just maybe) it would be nicer to refactor this check out into a helper method in DisplayLinkToSearchResult so that the information about types supported by sw360:DisplaySearchResultLink would by in only one place. But then you'd have to create a custom tld function for that.